src/openmc_fusion_benchmarks/benchmark.py

```diff
@@ -44,10 +46,65 @@ def build_tallies(self):
         """Build tallies for the benchmark."""
         pass
 
-    @abstractmethod
+    def _read_metadata(self):
+        """Read metadata from the benchmark specification."""
+        metadata = self._benchmark_spec['metadata']
+
+        lines = []
+        lines.append(f"📘 Title: {metadata.get('title', 'N/A')}")
+        lines.append("")
+        lines.append(f"🔖 Type: {metadata.get('type', 'N/A')}")
+        lines.append("")
+        lines.append(f"📂 Category: {metadata.get('category', 'N/A')}")
+        lines.append("")
+        lines.append(f"🧮 Version: {metadata.get('version', 'N/A')}")
+        lines.append("")
+        lines.append(f"📝 Description: {metadata.get('description', 'N/A')}")
+        lines.append(f"📅 Date: {metadata.get('date', 'N/A')}")
+
+        location = metadata.get("location", {})
+        lines.append("📍 Location:")
+        lines.append(f"   - Facility: {location.get('facility', 'N/A')}")
+        lines.append(f"   - City: {location.get('city', 'N/A')}")
+        lines.append(f"   - Country: {location.get('country', 'N/A')}")
+        lines.append("")
+
+        references = metadata.get("references", [])
+        lines.append("🔗 References:")
+        for ref in references:
+            lines.append(f"   - Title: {ref.get('title', 'N/A')}")
+            if 'doi' in ref:
+                lines.append(f"     DOI: {ref['doi']}")
+            if 'url' in ref:
+                lines.append(f"     URL: {ref['url']}")
+
+        authors = metadata.get("authors", [])
+        if authors:
+            lines.append("")
+            lines.append("👥 Authors:")
+            for author in authors:
+                name = author.get("name", "N/A")
+                affiliation = author.get("affiliation", "N/A")
+                email = author.get("email", "N/A")
+                lines.append(f"   - {name} ({affiliation}, {email})")
+
+        self._metadata = "\n".join(lines)
+
+    @property
     def metadata(self):
-        """Get metadata for the benchmark."""
-        pass
+        """Show metadata for the benchmark."""
+
+        class _MetadataDisplay:
```

Comment:
I don't think we need this nested class. If someone wants to display the metadata they should call `print()` which should handle the text contained in the `_metadata` attribute just fine.